
Key disagreement: dir_key 0011 != json_key 0010
wandb: Downloading large artifact train_256_standard:latest, 66.56MB. 1 files... Done. 0:0:0
Python 3.8.11 (default, Aug  6 2021, 09:57:55) [MSC v.1916 64 bit (AMD64)]
Type 'copyright', 'credits' or 'license' for more information
IPython 7.29.0 -- An enhanced Interactive Python. Type '?' for help.
PyDev console: using IPython 7.29.0
Traceback (most recent call last):
  File "C:\Users\acb6595\Anaconda3\envs\research_portable_wds_git\lib\site-packages\IPython\core\interactiveshell.py", line 3444, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-1-f07ec4451487>", line 1, in <module>
    val_split(0)
TypeError: 'list' object is not callable
Out[2]: ['a/airfield/00000001.jpg', '0']